File: Reserva-Hotel-com-Front-main/ProjetoCs-Reserva_de_Hotel-main/ReservaDeHotel/Models/Dono.cs
using System.ComponentModel.DataAnnotations;

namespace ReservaDeHotel.Models;
public class Dono
{
    [Key]
    public int IdDono {get;set;}
    public string? Nome { get; set;}
    public int NumeroTelefone { get; set;}
    public string? Email { get; set;}
    public string? Login { get; set;}
    public string? Senha { get; set;}
    public List<Hotel>? Hoteis { get; set;}
}

File: Reserva-Hotel-com-Front-main/ReservaHotelFront/src/app/Dono.ts
import { Hotel } from './Hotel';

export class Dono {
    donoId: number = 0;
    nome: string = '';
    numeroTelefone: number = 0;
    email: string = '';
    login: string = '';
    senha: string = '';
    hotelId: number = 0;
    hotel: Hotel | undefined;
}
